### PR TITLE
better explain the form type

### PR DESCRIPTION
**Pull request by dbu**
some documentation to go with https://github.com/symfony-cmf/SeoBundle/pull/277

### CODE REVIEW

File: bundles/seo/seo_aware.rst [hunk 1]
```diff
@@ -240,14 +240,29 @@ the ``SeoMetadata`` entity.
 Form Type
 ---------
 
-The bundle also provides a special form type called ``seo_metadata``. This
-form type can be used in forms to edit the ``SeoMetadata`` object.
+The bundle also provides a special form type called ``Symfony\Cmf\Bundle\SeoBundle\Form\Type\SeoMetadataType``
```
Comment (dbu) [inline bundles/seo/seo_aware.rst]: we should probably hunt through all of the documentation for these... but not in this PR

**Comment by dbu** [comment]
@WouterJ do you have an idea what the sphinx error is about? api_url indeed is assigned a string, so what is wrong? did they introduce api_url as a system variable and we somehow clash with that now?

File: bundles/seo/seo_aware.rst [hunk 2]
```diff
@@ -240,14 +240,29 @@ the ``SeoMetadata`` entity.
...
+(use ``seo_metadata`` for Symfony versions older than 2.8). This form type can be
+used in forms to edit the ``SeoMetadata`` object::
+
+    use Sonata\AdminBundle\Form\FormMapper;
+
+    /** @var FormMapper $formMapper */
+    $formMapper
+        ->add('seoMetadata', 'Symfony\Cmf\Bundle\SeoBundle\Form\Type\SeoMetadataType')
```
Comment (WouterJ) [inline bundles/seo/seo_aware.rst]: In the core docs, we decided to use `SeoMetadataType::class` + a use statement.

Comment (dbu) [inline bundles/seo/seo_aware.rst]: indeed much more elegant. but that is PHP 5.5 + only. is that ok?

Comment (dbu) [inline bundles/seo/seo_aware.rst]: ping @WouterJ

**Comment by dbu** [comment]
rebased to get the sphinx build fix

**Pull request merged**